internal/ethereum/ethereum.go

```diff
@@ -66,12 +66,27 @@ func NewEthereumConnector(ctx context.Context, conf config.Section) (cc ffcapi.A
 		eventBlockTimestamps:       conf.GetBool(EventsBlockTimestamps),
 		eventFilterPollingInterval: conf.GetDuration(EventsFilterPollingInterval),
 		traceTXForRevertReason:     conf.GetBool(TraceTXForRevertReason),
-		retry: &retry.Retry{
-			InitialDelay: conf.GetDuration(RetryInitDelay),
-			MaximumDelay: conf.GetDuration(RetryMaxDelay),
-			Factor:       conf.GetFloat64(RetryFactor),
-		},
+		retry:                      &retry.Retry{},
```

Comment:
Now that I see this I'm confused, it seems that this was the correct level for the connector.retry for the HTTP and WSS configuration but just because it turns out that it's being used for the block listener then we are completely deprecating the `connector.retry` section? and moving things up to `connector.blockListener`? I think we need to keep both